skimage/segmentation/random_walker_segmentation.py

```diff
@@ -428,8 +428,8 @@ def random_walker(data, labels, beta=130, mode='cg_j', tol=1.e-3, copy=True,
     # Parse input data
     if mode not in ('cg_mg', 'cg', 'bf', 'cg_j', None):
         raise ValueError(
-            f"{mode} is not a valid mode. Valid modes are 'cg_mg',"
-            f" 'cg', 'cg_j', 'bf', and None")
+            f'{mode} is not a valid mode. Valid modes are \'cg_mg\','
+            f' \'cg\', \'cg_j\', \'bf\', and None')
```

Comment:
Please revert this change as per my latest [comment](https://github.com/scikit-image/scikit-image/pull/5479#issuecomment-882111317).